scripts/foreach.sh

```diff
@@ -75,6 +75,20 @@ if [[ "${1:-}" == "--up" ]]; then
     shift
 fi
 
+down=""
+down_desc=""
+if [[ "${1:-}" == "--down" ]]; then
+    if [ ! -f package.json ]; then
+      echo "--down can only be executed from within a module directory (looking for package.json)"
+      exit 1
+    fi
+
+    scope=$(node -p "require('./package.json').name")
+    up=" --scope ${scope} --include-dependents"
+    up_desc="('${scope}' and its consumers)"
```

Comment:
copy pasta typo in variable names.

Might be worth defining just 2 vars - `direction` and `direction_desc` - and use the same variables for both up and down.